objects/src/accounts/stub.rs

```diff
@@ -9,9 +9,9 @@ use super::{hash_account, Account, AccountId, Digest, Felt};
 /// The [AccountStub] is composed of:
 /// - id: the account id ([AccountId]) of the account.
 /// - nonce: the nonce of the account.
-/// - vault_root: a commitment to the account's vault ([AccountVault]).
-/// - storage_root: accounts storage root ([AccountStorage]).
-/// - code_root: a commitment to the account's code ([AccountCode]).
+/// - vault_root: a commitment to the account's vault (\[AccountVault\]).
+/// - storage_root: accounts storage root (\[AccountStorage\]).
+/// - code_root: a commitment to the account's code (\[AccountCode\]).
```

Comment:
These supposed to be links to the corresponding structs. We should be able to fix these by doing something like `[super::AccountStorage]`.